operator/builtin/parser/xml/node.go

```diff
@@ -0,0 +1,73 @@
+package xml
+
+import (
+	"bytes"
+	"encoding/xml"
+)
+
+// Node represents an XML element
+type Node struct {
+	Type       string
+	Value      string
```

Comment:
We should double check that these are well established terms in the xml universe.

Reply:
It looks like they're not the default terms for this concept. Instead, tag and contents are what is often used. However, I'm not sure if those read better in regards to a log entry. What are your thoughts?

Reply:
I think we should defer to the domain's terminology, unless there's a need otherwise.

Reply:
The terminology has been switched to tag and content.